open-media-match/src/OpenMediaMatch/__init__.py

```diff
@@ -1 +1,30 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates.
+
+import flask
+
+
+def create_app():
+    """
+    Create and configure the Flask app
+    """
+    app = flask.Flask(__name__, instance_relative_config=True)
+
+    @app.route("/")
+    def ping():
+        """
+        Liveness/readiness check endpoint for your favourite Layer 7 load balancer
+        """
+        return "I-AM-ALIVE"
+
+    @app.route("/hash/image")
+    def hash_image():
+        """
+        Hash an image
+        """
+        image_url = flask.request.args.get('url')
+        if image_url is None:
+            flask.abort(400)
+
+        return f"If this was implemented you'd be seeing the PDQ hash of the image at {image_url} here"
```

Comment:
Following py-tx we want the hashing algorithms used to be configurable rather than just PDQ, it's just that PDQ is an ideal test hash for images.

Dumb question on API - some alternatives we could have done:
1. [What is proposed here] `hash_photo(ptr) -> PDQ`, `hash_video(ptr) -> ...` `hash_text(txt) -> ...`
2. `hash(content_type, ptr) -> dict[algo => result]`
3. `hash_syncronous(content_type, ptr) -> dict[algo => result]` `hash_async(cotent_type, ptr) -> dict[algo => result]`

What are your thoughts on APIs per content vs per return path vs per algo?

Reply:
I will probably chop this particular `@app.route` out. It's not really a proposal for anything concrete, more an example so we had _something_ that wasn't just a status/ping/helloworld.